test/contract/arbRollup.spec.ts

```diff
@@ -159,7 +159,7 @@ const setup = async () => {
   const bridgeCreatorFac = (await ethers.getContractFactory(
     'BridgeCreator'
   )) as BridgeCreator__factory
-  const bridgeCreator = await bridgeCreatorFac.deploy()
+  const bridgeCreator = await bridgeCreatorFac.deploy(117964)
```

Comment:
missing `maxDataSize` in `rollupCreator.createRollup`

Reply:
ty fixed